server/src/boot/environments/development.js

```diff
@@ -1,7 +1,9 @@
-import getNodeEnv from "../../config/getNodeEnv.js";
+ import getNodeEnv from "../../config/getNodeEnv.js";
```

Comment:
While making other updates, let's remove the space that was added here at the beginning of the line so this file remains untouched